base/fastmath.jl

```diff
@@ -124,6 +125,38 @@ function make_fastmath(symb::Symbol)
 end
 make_fastmath(expr) = expr
 
+"""
+@fastmath expr
```

Comment:
>No, just the signature, see the doc for other macros/functions

Gotcha.  Done. thanks!

See [CONTRIBUTING.md#improving-documentation](https://github.com/JuliaLang/julia/blob/d3db31200c09a99ad9f7cbc019dd0aa56c6944a7/CONTRIBUTING.md#improving-documentation)

Okay thanks. I made those changes, and I'm running `make docs` now. I'll post again when that's done.